ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.cpp

```diff
@@ -0,0 +1,221 @@
+// [WriteFile Name=BrowseOGCAPIFeatureService, Category=Layers]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "BrowseOGCAPIFeatureService.h"
+#include "OgcFeatureService.h"
+#include "OgcFeatureServiceInfo.h"
+#include "OgcFeatureCollectionTable.h"
+#include "OgcFeatureCollectionInfo.h"
+#include "FeatureLayer.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+
+#include<QQmlEngine>
+
+using namespace Esri::ArcGISRuntime;
+
+BrowseOGCAPIFeatureService::BrowseOGCAPIFeatureService(QObject* parent /* = nullptr */):
+    QObject(parent),
+    m_map(new Map(BasemapStyle::ArcGISTopographic, this))
+{
+    // Initialise OGC feature service
+    m_featureServiceUrl = "https://demo.ldproxy.net/daraa";
+    loadFeatureService(m_featureServiceUrl);
+}
+
+BrowseOGCAPIFeatureService::~BrowseOGCAPIFeatureService() = default;
+
+void BrowseOGCAPIFeatureService::init()
+{
+    // Register the map view for QML
+    qmlRegisterType<MapQuickView>("Esri.Samples", 1, 0, "MapView");
+    qmlRegisterType<BrowseOGCAPIFeatureService>("Esri.Samples", 1, 0, "BrowseOGCAPIFeatureServiceSample");
+}
+
+void BrowseOGCAPIFeatureService::handleError(Esri::ArcGISRuntime::Error error)
+{
+    if (error.additionalMessage().isEmpty())
+    {
+        setErrorMessage(error.message());
+    }
+    else
+    {
+        setErrorMessage(error.message() + "\n" + error.additionalMessage());
+    }
+    emit errorMessageChanged();
```

Comment:
you are already calling `emit errorMessageChanged` in the `setErrorMessage` function so this will fire twice